src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -373,6 +373,7 @@
     <Compile Include="$(BclSourcesRoot)\System\Variant.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
+    <Compile Include="$(BclSourcesRoot)\Interop\Unix\Interop.BOOL.cs" />
```

Comment:
Would it work to just reference the file from `shared`  instead of adding a new copy?

Reply:
Yeah, that would work too. Probably a better solution.